### PR TITLE
LPS-136236 Information Template Add and Edit

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-136236

### CODE REVIEW

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

File: modules/apps/template/template-web/src/main/java/com/liferay/template/web/internal/portlet/action/UpdateDDMTemplateMVCActionCommand.java [hunk 1]
```diff
@@ -98,11 +98,31 @@ else if (Objects.equals(smallImageSource, "file")) {
 		ServiceContext serviceContext = ServiceContextFactory.getInstance(
 			DDMTemplate.class.getName(), uploadPortletRequest);
 
-		DDMTemplate ddmTemplate = _ddmTemplateService.updateTemplate(
-			ddmTemplateId, classPK, nameMap, descriptionMap,
-			DDMTemplateConstants.TEMPLATE_TYPE_DISPLAY, StringPool.BLANK,
-			language, script, cacheable, smallImage, smallImageURL,
-			smallImageFile, serviceContext);
+		DDMTemplate ddmTemplate;
```
Comment (ealonso) [inline modules/apps/template/template-web/src/main/java/com/liferay/template/web/internal/portlet/action/UpdateDDMTemplateMVCActionCommand.java:101]: It should be:

DDMTemplate ddmTemplate = null;

cc: @lfbesada

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**